OOP/hw1.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+class Shape
+{
+    public $width;
+    public $height;
+    
+
+    function __construct($width, $height) {
+        $this->width = $width;
+        $this->height = $height;
+    }
+}
+
+class Triangele extends Shape
+{
+    function area()
+    {
+        return ($this->width * $this->height) / 2;
+    }
+}
+
+class Rectangle extends Shape
+{
+    function area()
+    {
+        return $this->width * $this->height;
+    }
+}
+
+// test
+$tri = new Triangle(10, 20);
```

Comment:
Because you set classname wrongly above, so that your source code doesn't work.
Please try to test your source code before pushing on github :)